pyp2spec/pyp2conf.py

```diff
@@ -27,12 +27,14 @@ class PackageNotFoundError(Pyp2specError):
 class PypiPackage:
     """Store and process the package data obtained from PyPI."""
 
-    def __init__(self, package_name, *, package_metadata=None, session=None):
+    def __init__(self, package_name, *, version=None, package_metadata=None, session=None):
         self.package_name = package_name
+        self._version = version
```

Comment:
Idea: Maybe the property is redundant? If the init calls this once instead, the "data flow" could be simpler.

```suggestion
        self.version = version or self._get_version_from_package_metadata()
```